lib/src/types.rs

```diff
@@ -66,16 +94,20 @@ pub enum Type {
     Char,
     String,
     Void,
+    Null,
+    Class(String),
```

Comment:
Oh good to have the class as type as well. Though there should probably be some documentation about this having to be the full class path(Though that is probably irrelevant outside of Javas Object type for our project)